tools/cli/test/e2e/cli/split_test.go

```diff
@@ -75,19 +86,105 @@ func TestSplitVersions(t *testing.T) {
 			cmd.Stderr = &e
 			require.NoError(t, cmd.Run(), e.String())
 
+			versions := getVersions(t, cliPath, base, folder)
 			for _, version := range versions {
+				if slices.Contains(skipVersions, version) {
+					continue
+				}
 				if tc.env == "prod" && version == "2025-01-01" {
 					continue
 				}
+				fmt.Printf("Validating version: %s\n", version)
 				validateFiles(t, version, folder)
 			}
 		})
 	}
 }
+
+func TestSplitVersionsForOASWithExternalReferences(t *testing.T) {
+	folder := "dev"
+	cliPath := NewBin(t)
+	base, err := filepath.Abs("../../data/split/" + folder + "/openapi-api-registry.json")
+	require.NoError(t, err)
+	copyMMSFileToOutput(t, folder)
+
+	cmd := exec.Command(cliPath,
+		"split",
+		"-s",
+		base,
+		"-o",
+		getOutputFolder(t, folder)+"/api-registry-output.json",
+		"--env",
+		folder,
+	)
+	var o, e bytes.Buffer
+	cmd.Stdout = &o
+	cmd.Stderr = &e
+	require.NoError(t, cmd.Run(), e.String())
+
+	versions := getVersions(t, cliPath, base, folder)
+	fmt.Printf("Versions: %v\n", versions)
+	for _, version := range versions {
+		if folder == "prod" && version == "2025-01-01" {
+			continue
+		}
```

Comment:
can we change this to be `if "prod" and date in the future?`

Reply:
yes! good point

Reply:
I'm opening another follow-up pr so I'll add this right away in https://github.com/mongodb/openapi/pull/153